buildSrc/src/main/kotlin/io/spine/internal/dependency/Spine.kt

```diff
@@ -40,62 +40,98 @@ object Spine {
      */
     object ArtifactVersion {
 
-        /** The version of [ProtoData]. */
-        @Deprecated("Please use `ProtoData.version` instead.")
-        const val protoData = ProtoData.version
-
-        /** The version of [Spine.base]. */
-        const val base = "2.0.0-SNAPSHOT.180"
+        /**
+         * The version of [Spine.base].
+         *
+         * @see <a href="https://github.com/SpineEventEngine/base">spine-base</a>
+         */
+        const val base = "2.0.0-SNAPSHOT.182"
 
-        /** The version of [Spine.reflect]. */
+        /**
+         * The version of [Spine.reflect].
+         *
+         * @see <a href="https://github.com/SpineEventEngine/reflect">spine-reflect</a>
+         */
         const val reflect = "2.0.0-SNAPSHOT.182"
 
-        /** The version of [Spine.logging]. */
-        const val logging = "2.0.0-SNAPSHOT.191"
+        /**
+         * The version of [Spine.logging].
+         *
+         * @see <a href="https://github.com/SpineEventEngine/logging">spine-logging</a>
+         */
+        const val logging = "2.0.0-SNAPSHOT.186"
```

Comment:
Again, this is a downgrade. Please review.

Reply:
@yevhenii-nadtochii, the most recent version which can be used here is `2.0.0-SNAPSHOT.191`. After that the build complains about the lack of `flogger:api` dependency:

```
* What went wrong:
Could not determine the dependencies of task ':extractIncludeProto'.
> Could not resolve all files for configuration ':compileProtoPath'.
   > Could not find flogger:api:.
     Required by:
         project : > io.spine:spine-logging:2.0.0-SNAPSHOT.192 > io.spine:spine-logging-jvm:2.0.0-SNAPSHOT.192
```